tests/dummy_opt/config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "_name_or_path": "facebook/opt-125m",
+  "activation_dropout": 0.0,
+  "activation_function": "relu",
+  "architectures": [
+    "MyOPTForCausalLM"
```

Comment:
previously, we are hacking `OPTForCausalLM`. this pr changes it to the normal one, i.e. oot registration models have its own architecture.